torchvision/ops/ps_roi_align.py

```diff
@@ -19,23 +19,26 @@ def ps_roi_align(
     mentioned in Light-Head R-CNN.
 
     Args:
-        input (Tensor[N, C, H, W]): input tensor
+        input (Tensor[N, C, H, W]): The input tensor, i.e. a batch with ``N`` feature maps
```

Comment:
This is actually not correct -- N is the batch size, C is the number of feature maps (aka channels)